index.js

```diff
@@ -4,9 +4,11 @@ var chokidar = require('chokidar');
 var xtend = require('xtend');
 
 module.exports = watchify;
-module.exports.args = {
-    cache: {}, packageCache: {}
+module.exports.defaults = function (opts) {
+    return xtend({cache: {}, packageCache: {}}, opts);
 };
+// DEPRECATED - will be removed in a future version
+module.exports.args = watchify.defaults();
```

Comment:
I'm not a fan of magical getters/setters. I don't think I want to do away with `args` just yet. (notice how old this PR is)